src/lambda_function.py

```diff
@@ -1,20 +1,19 @@
 import time
 
 from webdriver_wrapper import WebDriverWrapper
+from selenium.webdriver.common.keys import Keys
 
 
 def lambda_handler(*args, **kwargs):
     driver = WebDriverWrapper()
 
     driver.get_url('https://www.google.es/')
 
-    driver.set_input_value('//input[@id="lst-ib"]', '21 buttons')
-
-    driver.click('//center//img[@alt="Google"]')
-    time.sleep(0.5)
```

Comment:
You cannot delete the sleeps and not replace them by `awaits` because the website wont be loaded at the time you do the click.

I've tried this code and without the `sleep` it's not working

Reply:
Yes, a little sleep is definitely needed, will add.